GalaxyZooTouchTable/GalaxyZooTouchTable/Lib/CommonData.cs

```diff
@@ -0,0 +1,50 @@
+﻿using GalaxyZooTouchTable.Models;
+using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+
+namespace GalaxyZooTouchTable.Lib
+{
+    public class CommonData : INotifyPropertyChanged
+    {
+        private static CommonData _instance = null;
+        public ObservableCollection<TableUser> AllUsers { get; set; } = new ObservableCollection<TableUser>();
+        public TableUser PersonUser = new PersonUser();
+        public TableUser LightUser = new LightUser();
+        public TableUser StarUser = new StarUser();
+        public TableUser HeartUser = new HeartUser();
+        public TableUser FaceUser = new FaceUser();
+        public TableUser EarthUser = new EarthUser();
+
+        protected CommonData()
+        {
+            PopulateUsers();
+        }
+
+        public static CommonData GetInstance()
+        {
+            if (_instance == null)
+                _instance = new CommonData();
+
+            return _instance;
+        }
+
+        private void PopulateUsers()
+        {
+            AllUsers.Add(PersonUser);
```

Comment:
Instead of having the user classes be member variables, it may make more sense to just initialize and add them to AllUsers here like: 

`AllUsers.Add(new PersonUser());`

I think you should do this if you aren't ever planning on referencing the individual user classes

Reply:
I have it set up this way because the PersonUser added to `AllUsers` should be the same as the one that is created when `ClassificationPanelViewModel` is instantiated with a PersonUser. 

Because of this, I create the global PersonUser first in this constructor, and refer to that instance for the addition to `AllUsers` as well as the `CommonData.GetInstance().PersonUser` that is passed to the `ClassificationPanelViewModel` constructor. This is necessary when doing some comparison of users in the `NotificationsViewModel`.